### ISSUE TITLE
Improved it's UI design

### ISSUE THREAD

**Issue by Pranab009** [itsdrnoob/DataMonitor#43]
**Is your feature request related to a problem? Please describe.**

Improved it's UI design.
See images many texts are not visible or hidden.and don't need to swipe up.(I mean one page)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
I loved this app, many improvements are needed. 
I know this is in early stage. 
See this images
![Screenshot_20220731-141238_1](https://user-images.githubusercontent.com/65340361/188271383-dc66d67e-6d20-4b95-ac17-453733206b01.png)

![Screenshot_20220902-111037_1](https://user-images.githubusercontent.com/65340361/188271387-cc811ca0-2afb-4307-8690-4485d32d351f.png)

![Screenshot_20220902-143310](https://user-images.githubusercontent.com/65340361/188271388-007b3e05-e4bd-420c-81db-52edd4bc9660.png)

![Screenshot_20220902-143610](https://user-images.githubusercontent.com/65340361/188271390-2f0cb84e-2e28-40e4-8c85-c47efc582b5d.png)

![Screenshot_20220902-143505](https://user-images.githubusercontent.com/65340361/188271396-13d511f4-ea8e-4402-8808-8b0881c5738c.png)

![Screenshot_20220902-154312](https://user-images.githubusercontent.com/65340361/188271398-3aa2ad58-5b55-49c5-801b-0dcee27c0fb0.png)

![Screenshot_20220902-160227](https://user-images.githubusercontent.com/65340361/188271399-9947b097-613f-401f-ab2c-b264c8f07d74.png)

See this
![DocScanner 02-Sep-2022 2-36 pm (1)](https://user-images.githubusercontent.com/65340361/188271402-321b83e4-da26-4bd3-a2c4-df62e24de466.jpg)

![Screenshot_20220902-160227_1](https://user-images.githubusercontent.com/65340361/188271403-da4e40ec-c4d6-461a-83f5-832ffb5fc7b0.png)

**Comment by itsdrnoob** [comment]
Agreed! Improvements to support different screen densities are still to be done.
In the meantime, if you need a quick-fix, try increasing the screen dpi.
